### PR TITLE
Postpone policy recommendation scope watches until apiserver is up an…

### PR DESCRIPTION
**Pull request by rene-dekker**
Affects: EV-3878 EV-3861

### CODE REVIEW

**Review by dimitri-nicolo — APPROVED** [review]
lgtm

File: pkg/controller/policyrecommendation/policyrecommendation_controller.go [hunk 1]
```diff
@@ -65,6 +65,7 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 	}
 	licenseAPIReady := &utils.ReadyFlag{}
 	tierWatchReady := &utils.ReadyFlag{}
+	policyRecScopeWatchReady := &utils.ReadyFlag{}
```
Comment (tmjd) [inline pkg/controller/policyrecommendation/policyrecommendation_controller.go:68]: It seems like we should check this ready flag (like we do with the license and tier ones) before fetching the PolicyRecommendationScopes in the Reconcile.

Comment (rene-dekker) [reply pkg/controller/policyrecommendation/policyrecommendation_controller.go:68]: Done, thank you.

**Review by tmjd — APPROVED** [review]
LGTM

**Pull request merged**